src/main_thread/decrypt/utils/clean_old_loaded_sessions.ts

```diff
@@ -28,18 +30,44 @@ import type LoadedSessionsStore from "./loaded_sessions_store";
  */
 export default async function cleanOldLoadedSessions(
   loadedSessionsStore: LoadedSessionsStore,
+  activeRecords: KeySessionRecord[],
   limit: number,
 ): Promise<void> {
   if (limit < 0 || limit >= loadedSessionsStore.getLength()) {
     return;
   }
   log.info("DRM: LSS cache limit exceeded", limit, loadedSessionsStore.getLength());
   const proms: Array<Promise<unknown>> = [];
```

Comment:
Maybe, but here the return type was not important because never relied on so I typed it as `unknown` so it is not an obstacle now or in future evolutions.